doc/lua_api.txt

```diff
@@ -5947,6 +5949,18 @@ object you are working with still exists.
 
 * `get_pos()`: returns `{x=num, y=num, z=num}`
 * `set_pos(pos)`: `pos`=`{x=num, y=num, z=num}`
+* `get_velocity()`: returns the velocity, a vector.
+* `add_velocity(vel)`
+    * `vel` is a vector, e.g. `{x=0.0, y=2.3, z=1.0}`
+    * In comparison to using get_velocity, adding the velocity and then using
+      set_velocity, add_velocity is supposed to avoid synchronization problems.
```

Comment:
should include a note that set_velocity does not exist for players at all

Reply:
Is this comment even true? I don't see how there's any synchronisation problems here, the server is in control of velocity

Reply:
It's not:
https://github.com/minetest/minetest/blob/d80def5bbf58c9ddf1ab3d62e2e456b8dc824c98/src/network/serverpackethandler.cpp#L489

I'm also not aware of any packet that would tell the client to set a specific velocity.